common/djangoapps/auth_exchange/tests/test_forms.py

```diff
@@ -12,10 +12,12 @@
 from provider import scope
 import social.apps.django_app.utils as social_utils
 
-from auth_exchange.forms import AccessTokenExchangeForm
-from auth_exchange.tests.utils import AccessTokenExchangeTestMixin
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixinFacebook, ThirdPartyOAuthTestMixinGoogle
 
+from ..forms import AccessTokenExchangeForm
```

Comment:
Is there a reason we cannot use absolute imports here? If so, please add a comment with that reason.